ArjunaJTS/standalone/start.sh

```diff
@@ -74,17 +58,15 @@ if test "x$HOST_ADDRESS" = "x"; then
   exit 1
 fi
 if [ $# -lt 2 ]; then
-  echo "syntax {jdkorb|jacorb} {NS | TM | RM | CL}"
+  echo "syntax {NS | TM | RM | CL}"
```

Comment:
I think this is good but I think the test `if [ $# -lt 2 ]; then` should be changed to `if [ $# -lt 1 ]; then` and I think lower down `ORBTYPE=$1` should be removed (and I think `#ORBTYPE=openjdkorb` should be deleted) and I think lower done `case $2 in` should be `case $1 in`